zerver/openapi/zulip.yaml

```diff
@@ -5278,8 +5278,13 @@ paths:
             subscribed to appropriate streams or use a shared history
             search narrow with this endpoint.
 
-            **Changes**: In Zulip 2.1.0, added support for using user/stream IDs
-            when constructing narrows for a message's sender, its stream and/or
+            **Changes**: In Zulip 7.0 (feature level 177), support was
+            added for three narrows related to direct messages: `is:dm`,
```

Comment:
similarly, s/three narrows/three filters/, here and a few places below

Reply:
Done. Also made sure the parameter description included "filter" so that the changes note would read clearly.